app/Http/Requests/Campaign/CreateRequest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace App\Http\Requests\Campaign;
+
+use App\Models\Campaign;
+use Illuminate\Foundation\Http\FormRequest;
+
+class CreateRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
```

Comment:
This should already be being handled via the resource controller https://github.com/thybag/RPGCampaignManager/pull/28/files#diff-7c9703676fae3d99b89b510ef1f5fee516141b0c72303e90b24350d492e98652R16 (basically it'll auto call the policy based on resource controller route name) meaning this should be taken care of ahead of time.

Reply:
@thybag I am not sure if this file is still necessary since you've added a request on the master branch?